scenarios/scenario-18/swagger.yaml

```diff
@@ -0,0 +1,32 @@
+#Copyright (c)  WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# WSO2 Inc. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+---
+openapi: 3.0.0
+info:
+  title: hello-world
+  version: v1
+  description: This API contains operations related to node service.
+x-wso2-basePath: /node/1.0.0
+x-wso2-production-endpoints: hello-world-service
+x-wso2-mode: privateJet
+paths:
+  /hello/node:
+    get:
+      responses:
+        "200":
+          description: ""
+      x-auth-type: None
+      x-throttling-tier: Unlimited
```

Comment:
We dont need to set these to extensions.. Let's remove those.

Reply:
extra extensions removed